homeassistant/components/powerwall/translations/en.json

```
@@ -1,19 +1,23 @@
 {
     "config": {
         "abort": {
-            "already_configured": "Device is already configured"
+            "already_configured": "Device is already configured",
+            "reauth_successful": "Re-authentication was successful"
         },
         "error": {
             "cannot_connect": "Failed to connect",
+            "invalid_auth": "Invalid authentication",
             "unknown": "Unexpected error",
             "wrong_version": "Your powerwall uses a software version that is not supported. Please consider upgrading or reporting this issue so it can be resolved."
         },
         "flow_title": "Tesla Powerwall ({ip_address})",
         "step": {
             "user": {
                 "data": {
-                    "ip_address": "IP Address"
+                    "ip_address": "IP Address",
+                    "password": "Password"
                 },
+                "description": "The password is usually the last 5 characters of the serial number and can be found in the Telsa app.",
```

https://www.tesla.com/en_gb/support/energy/powerwall/own/monitoring-from-home-network - 
Backup Gateway: Last five characters of the Gateway serial number.
Backup Gateway 2: Last five digits of characters on the sticker.
```suggestion
                "description": "The password is usually the last 5 characters of the serial number for Backup Gateway and can be found in the Telsa app; or the last 5 characters of the password found inside the door for Backup Gateway 2.",
```